packages/backend/src/models/DashboardModel/DashboardModel.ts

```diff
@@ -321,10 +322,27 @@ export class DashboardModel {
             .where({ dashboard_version_id: versionId.dashboard_version_id });
     }
 
+    // TODO: This can be removed once we can have a semantic layer as a project connection
+    private async getDashboardVersionTileTypes(dashboardVersionId: number) {
+        const tileTypes = await this.database(DashboardTilesTableName)
+            .select<
+                {
+                    type: DashboardTileTypes;
+                }[]
+            >(`${DashboardTilesTableName}.type`)
+            .where(
+                `${DashboardTilesTableName}.dashboard_version_id`,
+                dashboardVersionId,
+            )
+            .distinctOn(`${DashboardTilesTableName}.type`);
+
+        return tileTypes.map((t) => t.type);
+    }
```

Comment:
you can actually use all the versions and users never be able to mix dashboard chart types even between versions (solves scheduled deliveries issue) also, it does not matter so feel free to ignore it!